scripts/generate-framework-agreement-counterpart-signature-pages.py

```diff
@@ -58,8 +61,10 @@
     args = docopt(__doc__)
 
     framework_slug = args['<framework_slug>']
-    framework_kwargs = declaration_definite_pass_schema = json.load(open(args["<framework_json>"], "r"))
     client = DataAPIClient(get_api_endpoint_from_stage(args['<stage>']), args['<api_token>'])
+    framework = client.get_framework(framework_slug)['frameworks']
+    framework_kwargs = framework['frameworkAgreementDetails']
+    framework_kwargs['frameworkName'] = framework['name']
```

Comment:
My thinking was just that using a regular framework object would be the most no-surprises route. Any developer coming to the template just gets this *"I know this - this is UNIX"* feel. Agree that it's more typing though.